package.json

```diff
@@ -18,8 +18,5 @@
     "pxt": "*",
     "tslint": "^6.1.1",
     "typescript": "^3.8.3"
-  },
```

Comment:
This will be added by the pxt-setup. No reason to commit it.

Reply:
But then it leaves the `package.json` as modified